datadog_checks_dev/datadog_checks/dev/tooling/signing.py

```diff
@@ -8,16 +8,16 @@
 
 # How long ddev will wait for GPG to finish, especially when asking dev for signature.
```

Comment:
`datadog_checks/dev/tooling/signing.py` is ignore in `pyproject.toml`, but looks like skip_glob is not working anymore with isort v5, see: https://github.com/PyCQA/isort/issues/1468